### PR TITLE
added sources from H-M

### PR DESCRIPTION
**Pull request by thja**

### CODE REVIEW

File: sources/harvest.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "HARVEST",
+  "name": "Harvest",
+  "categories": ["TOOLS"],
+  "organization": "APACHE",
```
Comment (pfrisella) [inline sources/harvest.json]: organization should be HARVEST

File: sources/harvest.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  "iconUrl": "https://adverity-gds-static.s3.amazonaws.com/icons/hadoop.png",
```
Comment (pfrisella) [inline sources/harvest.json]: iconUrl is incorrect

File: sources/harvest.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  "sourceUrl": "https://www.getharvest.com/",
+  "dataVisibility": ["PRIVATE", "PUBLIC"]
```
Comment (pfrisella) [inline sources/harvest.json]: is there PUBLIC data?

File: sources/ironsrc.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/ironsrc.json]: This source is already defined, you can remove this entry and org and use existing:
https://github.com/googledatastudio/ds-data-registry/blob/master/sources/ironsource.json

File: organizations/ironsrc.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/ironsrc.json]: This org is already defined, you can remove this entry and source and use existing:
https://github.com/googledatastudio/ds-data-registry/blob/master/organizations/ironsource.json

File: sources/moat.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "MOAT",
+  "name": "Moat",
+  "categories": ["ANALYTICS"],
+  "organization": "MOAT",
```
Comment (pfrisella) [inline sources/moat.json]: Moat is part of ORACLE

File: sources/moat.json [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/moat.json]: should it be Moat Analytics?

File: organizations/moat.json [hunk 8]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/moat.json]: If moat source is updated to use ORACLE org then you an remove this entry.

**Comment by thja** [comment]
@pfrisella please review again

**Pull request merged**